src/Core/KeyManagement/Models/Data/SignatureKeyPairData.cs

```diff
@@ -0,0 +1,11 @@
+﻿#nullable enable
+using Bit.Core.Enums;
+
+namespace Bit.Core.KeyManagement.Models.Data;
+
+public class SignatureKeyPairData
+{
+    required public SignatureAlgorithm SignatureAlgorithm { get; set; }
```

Comment:
⛏️ I think we usually swap `required` and `public`.

Reply:
Fixed in https://github.com/bitwarden/server/pull/5906/commits/9e0cdf3383ce93dc86bc93b6a3925ce4b04a5097